recipes-devtools/python/python3-labgrid.inc

```diff
@@ -3,6 +3,9 @@ HOMEPAGE = "https://github.com/labgrid-project"
 LICENSE = "LGPL-2.1-or-later"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=c0e9407a08421b8c72f578433434f0bd"
 
+PACKAGE_BEFORE_PN += "${PN}-coordinator"
+RDEPENDS:${PN}-coordinator = "${PN}"
+
```

Comment:
Why does this need to be a separate package? Simply installing the systemd service file should not hurt.

Reply:
I wanted to reduce the amount of systemd-services, that get started by default and not have it in there, but be a deliberate decision to activate the coordinator on a device.

Reply:
`labgrid-coordinator.service` is not started by default, hence installing the coordinator entry point and the systemd service does not change anything. A `systemctl enable labgrid-coordinator.service` is still needed.